istioctl/cmd/add-to-mesh_test.go

```diff
@@ -126,13 +126,19 @@ var (
 func TestAddToMesh(t *testing.T) {
 	cases := []testcase{
 		{
-			description:       "Invalid command args",
+			description:       "Invalid command args -missing service name",
```

Comment:
Add space between `-` and `missing` similar to line 135.

Reply:
fixed